step-definitions/TestSD/DSQ3SD.js

```diff
@@ -0,0 +1,13 @@
+const { Given, Then, When } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const DSHomepage = require("../../POM/TestPOM/DSHome");
+
+const dsHome = new DSHomepage
+
+Given(/^I am on darksky landing page$/, async function() {
+    await browser.url('https://www.darksky.net/')
```

Comment:
Once you define a step in a SD file, you don't need to  define it again on the same project.